File: python/fizzbuzz.py
def fizzbuzz(input_number):
	if input_number % 5 == 0 and input_number % 3 == 0:
		print("FizzBuzz")
	elif input_number % 5 == 0:
		print("Fizz")
	elif input_number % 3 == 0:
		print("Buzz")
	else:
		print(str(input_number))


if __name__ == "__main__":
	while True:
		input_value = input("Enter a number: ")
		fizzbuzz(int(input_value))

File: c/fizzbuzz.c
#include "stdio.h"

void fizzbuzz(int input){
	if(input % 5 == 0 && input % 3 == 0){
		printf("FizzBuzz\n");
	} else if(input % 5 == 0) {
		printf("Fizz\n");
	} else if(input % 3 == 0) {
		printf("Buzz\n");
	} else {
		printf("%d\n", input);
	}
}

int main() {
	int input_number;
	while(1){
		printf("Enter a number: ");
		scanf("%d", &input_number);
		fizzbuzz(input_number);
	}
	return 0;
}
